### PR TITLE
Responsible Body - Account page

### PR DESCRIPTION
**Pull request by kenfodder**
Responsible body homepage.

Todo: 

- ~~Tests~~

### CODE REVIEW

**Review by nigel-lowry — APPROVED** [review]

File: app/views/responsible_body/home/_closed_banner_multiple.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<p class="govuk-body">
+  From <span class="app-no-wrap">September 2020</span> to <span class="app-no-wrap">July 2021</span>, your schools and colleges in your <%= @responsible_body.humanized_type -%> received:
```
Comment (ralph-hawkins) [inline app/views/responsible_body/home/_closed_banner_multiple.html.erb:2]: can you remove the 'your' before 'schools' please?

File: app/views/responsible_body/home/_summary_reset_devices.html.erb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+<%# BOTH %>
+<% if @has_ordered_std_devices && @has_ordered_coms_devices %>
+  <p class="govuk-body">
+    Your devices may need to be reset soon:
+  </p>
+
+  <ul class="govuk-list govuk-list--bullet">
+    <li>Windows laptops and tablets need to be reset before <span class="app-no-wrap">30 September 2021</span>.</li>
+    <li>Huawei routers need to be reset before <span class="app-no-wrap">16 July 2021</span>.</li>
```
Comment (ralph-hawkins) [inline app/views/responsible_body/home/_summary_reset_devices.html.erb:34]: can you remove the full stops from the ends of these two list items please?

File: app/views/responsible_body/home/_summary_reset_devices.html.erb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+<%# JUST STD DEVICES %>
+<% if @has_ordered_std_devices && !@has_ordered_coms_devices %>
+  <p class="govuk-body">
+    Windows laptops and tablets need to be reset before <span class="app-no-wrap">30 September 2021</span>.
+  </p>
+
+  <ul class="govuk-list govuk-list--spaced">
+    <li><%= govuk_link_to "How to reset Windows laptops and tablets" , devices_guide_to_resetting_windows_laptops_and_tablets_path %></li>
+  </ul>
```
Comment (ralph-hawkins) [inline app/views/responsible_body/home/_summary_reset_devices.html.erb:11]: This could just be a paragraph as its a single item

**Review by ralph-hawkins — DISMISSED** [review]
Looks great, couple of minor things

**Comment by kenfodder** [comment]
@ralph-hawkins I also removed the full stops, made the <li> a <p>, and removed 'your' for the school HP as well since it's almost the same.

**Pull request merged**